### ISSUE TITLE
Not installing with new new capacitor/core

### ISSUE THREAD

**Issue by felipecarrillo100** [capacitor-community/sqlite#268]
**Describe the bug**
Recently a new ionic  capacitor core was published.  
When I do 

npm install @capacitor-community/sqlite it fails

**To Reproduce**
Install latest ionic 

1. Use ionic start to create a new project,  this will install the latest core
    "@capacitor/core": "3.5.1",
2. Install latest version of  sqlite plugin
npm install @capacitor-community/sqlite

**Expected behavior**
npm install will fail with this error
```
PS C:\node\ionic\spatialite> npm install @capacitor-community/sqlite@3.4.2
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: spatialite@0.0.1
npm ERR! Found: @capacitor/core@3.5.1
npm ERR! node_modules/@capacitor/core
npm ERR!   @capacitor/core@"3.5.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @capacitor/core@"~3.4.2" from @capacitor-community/sqlite@3.4.2
npm ERR! node_modules/@capacitor-community/sqlite
npm ERR!   @capacitor-community/sqlite@"3.4.2" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!

```

Note:  If I downgradet capacitor core to :      "@capacitor/core": "3.4.3",  it works fine again

**Comment by jepiqueau** [comment]
@felipecarrillo100 this is right you have to downgrade to capacitor 3.4.3. I usually works with the previous release of capacitor to allow for bug fixes. Here i am 2 releases behind as i worked  on the delete implementation for synchronisation  which had a huge impact on the code. Hope you will understand.

**Comment by felipecarrillo100** [comment]
Thanks for your reply.  Yes I understand of course. I opened the issue just to let you know there is a new capacitor/core and to let the users know there is a work around until you have time to catch up.  

Thanks also for the great work, your sqlite plugin is great.  I only wish it had Spatiality support, then it would be perfect for my needs.

Keep up the good work

**Comment by jepiqueau** [comment]
@felipecarrillo100 i looked a couple of time at it but i did not find a way. For me the best will be to write a wapper in javascript which use standard plugin command. I will not do that myself as i do not know Spatialite and when i ask ealier to some one asking for it to write a spec hé did not do it

**Comment by robingenz** [comment]
Hi @jepiqueau,
is there a reason why you do not allow the following?
```json
"peerDependencies": {
  "@capacitor/core": "^3.0.0"
},
``` 
There will be no breaking changes in minor and patch releases, so this should not be a problem, or?

**Comment by jepiqueau** [comment]
@robingenz i will review this later to day

**Comment by JarJarBoinks** [comment]
Hi all, I'm really new to both Ionic and Capacitor!
How would one downgrade their version of capacitor to work with the sqlite plugin?

**Comment by jepiqueau** [comment]
@JarJarBoinks i do this developement on my free time. You can ask the community i answer queries fast enough.

**Comment by felipecarrillo100** [comment]
> @JarJarBoinks what do you mean by downgrade. This plugin has always follow the capacitor/core release this time it has not as i was busy working on a fix which took longer than foreseen. I said also that i will have a lookto nigh

You downgrade the same way as you do with any NPM project.  Just specify the version in your package.json file.  You can google that.

**Comment by jepiqueau** [comment]
@robingenz thanks for your proposal @felipecarrillo100 @JarJarBoinks  This has been implemented in release 3.5.0. i will close the issue. Good luck with your development.

**Comment by robingenz** [comment]
Thank you very much @jepiqueau ! 🎉
